eng/helix/payload/dotnettest.cmd

```diff
@@ -0,0 +1,33 @@
+@echo off
+setlocal
+
+set testAssembly=%1
+set configuration=%2
+set targetFramework=%3
+set architecture=%~4
+
+set filterArgs=
+if not "%~5" == "" (
+   set filterArgs=--filter ^"%~5^"
+)
+
+set exit_code=0
+
+echo "Start tests..."
+
+dotnet.exe test ^
+  "%HELIX_CORRELATION_PAYLOAD%\%testAssembly%\%configuration%\%targetFramework%\%testAssembly%.dll" ^
+  --logger:"console;verbosity=normal" ^
+  --logger:"trx;LogFileName=%testAssembly%_%targetFramework%_%architecture%.trx" ^
+  --logger:"html;LogFileName=%testAssembly%_%targetFramework%_%architecture%.html" ^
+  --ResultsDirectory:%HELIX_WORKITEM_UPLOAD_ROOT% ^
+  --blame "CollectHangDump;TestTimeout=15m" ^
```

Comment:
Can this be parameterized so that it can be shared among all types of test runs?